Northstar.Client/mod/scripts/vscripts/ui/menu_mod_settings.nut

```diff
@@ -445,7 +449,7 @@ void function UpdateList()
 	}
 	file.updatingList = false
 
-	if ( file.conVarList.len() <= 0 )
+	if ( file.conVarList.len() <= 6 )
```

Comment:
I assume this change was to make the "go download some mods" text show up now that Northstar itself is adding some settings right?

Reply:
@ASpoonPlaysGames yes